packages/core/getUserAvailability.ts

```diff
@@ -164,7 +164,7 @@ export async function getUserAvailability(
     let startDate = dayjs(dateFrom);
     const endDate = dayjs(dateTo);
     while (startDate.isBefore(endDate)) {
-      dates.push(startDate.add(1, "day"));
+      dates.push(startDate);
```

Comment:
This added a day to all dates, which primarily showed itself by breaking the day booking limit but probably had knock-on effects with week/month/year.